### DISCUSSION TITLE
Allow to change names of components in temperature graph

### DISCUSSION THREAD

**Discussion by JuanR3D** [mainsail-crew/mainsail#2023] [General]
The idea is to allow to change the names of some elements (such as extruder, heater_bed and others) in order that these show in this graph with a different name than it's config name. 
For example, in my case, i want my machine elements translated to spanish, so i'd like to see "Extrusor" instead of "Extruder", or the name i want in that case.

**Comment by meteyou** [comment]
Thx for you post. There was already a PR for this for a short time. I wrote a detailed description why we will not do something like that in it.

Here is the link: https://github.com/mainsail-crew/mainsail/pull/1948#issuecomment-2234040027

Edit: I would rather add an option to translate also Klipper object names instead of user based alias names. But both are not really good solutions in my eyes...

**Reply by JuanR3D** [reply]
Hi! Thank you so much! 
I apologize, i didn't see that post.
